packages/a11y/src/add-intro-text.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Build the explanatory text to be placed before the aria live regions.
+ *
+ * This text is initially hidden from assistive technologies by using a `hidden`
+ * HTML attribute which is then removed once a message fills the aria-live regions.
+ *
+ * @return {HTMLParagraphElement} The explanatory text HTML element.
+ */
+export default function addIntroText() {
+	const introText = document.createElement( 'p' );
+
+	introText.id = `a11y-speak-intro-text`;
+	introText.className = 'a11y-speak-intro-text';
```

Comment:
Any specific reason for the backtick quotes, especially in contrast with the second line?

```suggestion
	introText.id = 'a11y-speak-intro-text';
	introText.className = 'a11y-speak-intro-text';
```

Reply:
Typo :)